# HG changeset patch
# User Pantxo Diribarne <pantxo.diribarne@gmail.com>
# Date 1519994805 -3600
#      Fri Mar 02 13:46:45 2018 +0100
# Node ID 735175432a20dcc373f30b0b09d149160d17e4b1
# Parent  bdec5e695ec32f33aafbda4e7d7572e552461dd8
getframe.m: update %!test for new offscreen system (bug #53186)

diff --git a/scripts/image/getframe.m b/scripts/image/getframe.m
--- a/scripts/image/getframe.m
+++ b/scripts/image/getframe.m
@@ -157,26 +157,26 @@ endfunction
 %! set (hax1, "units", "normalized");
 %! rect = [pos(1:2) pos(3:4)/2];
 %!
 %! frame = getframe (hax1, rect);
 %! hax2 = subplot (2,1,2);
 %! image (frame.cdata);
 %! title ("Lower left hand corner");
 
-%!testif HAVE_OSMESA <44338>
+%!testif HAVE_QOFFSCREENSURFACE
 %! hf = figure ("visible", "off");
 %! unwind_protect
 %!   pos = get (hf, "position");
 %!   assert (size (getframe (hf).cdata)(1:2), pos(4:-1:3));
 %! unwind_protect_cleanup
 %!   close (hf);
 %! end_unwind_protect
 
-%!testif HAVE_OSMESA <44338>
+%!testif HAVE_QOFFSCREENSURFACE
 %! hf = figure ("visible", "off");
 %! unwind_protect
 %!   hax = axes ("visible", "off", "position", [0 0 1 1]);
 %!   verts = [0 0; .5 0; 1 0; ...
 %!            0 .5; .5 .5; 1 .5; ...
 %!            0 1; .5 1; 1 1];
 %!   faces = [1 2 5 4; 2 3 6 5; 4 5 8 7; 5 6 9 8];
 %!   fvc = [1 0 0; 0 1 0; 0 0 1; 1 0 1];
